soco/core.py

```diff
@@ -1528,9 +1483,9 @@ def get_favorite_radio_shows(self, start=0, max_items=100):
         """Get favorite radio shows from Sonos' Radio app.
 
         Returns:
-        A list containing the total number of favorites, the number of
-        favorites returned, and the actual list of favorite radio shows,
-        represented as a dictionary with `title` and `uri` keys.
+            A list containing the total number of favorites, the number of
```

Comment:
I suspected that this was wrong, so I checked the code. That first "list" should be "dict". But shouldn't that return type be documented properly. I guess these have disappeared anyway with one of your other PR's